<launch>

	<include file = "$(find husky_gazebo)/launch/husky_empty_world.launch">
    	<arg name = "world_name" value = "$(find husky_highlevel_controller)/worlds/singlePillar.world"/>
    	<arg name = "laser_enabled" default = "true"/>
	</include>
	<node name = "StartStopServer" pkg = "husky_highlevel_controller" type = "StartStopServer"/>
	<node name = "tele_op" pkg = "teleop_twist_keyboard" type = "teleop_twist_keyboard.py" output = "screen"/>
	<node name = "StartStopClient" pkg = "husky_highlevel_controller" type = "StartStopClient"/>
	<node name = "husky_highlevel_controller" pkg = "husky_highlevel_controller" type = "husky_highlevel_controller" output = "screen">
    	<rosparam command = "load" file = "$(find husky_highlevel_controller)/config/default.yaml"/>
  	</node>
	<node name = "rviz" pkg = "rviz" type = "rviz"/>
</launch>
